src/components/PatientProfilePage/AmbulanceTransportSelector.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Typography from '@material-ui/core/Typography';
+import { Ambulance } from '../../graphql/queries/ambulances';
+
+const AmbulanceTransportSelector = ({
```

Comment:
is it worth spinning this off into a common component? (doesn't necessarily have to be done in this PR)

Reply:
I don't think so. There's already a common selector component `RadioSelector.tsx` for when the options' value and label are the same, I made `AmbulanceTransportSelector.tsx` and `HospitalTransportSelector.tsx` specifically for hospital and ambulance objects, because the value of the options are their id's, while the labels are either hospital.name or ambulance.vehicleNumber